mbuild/formats/lammpsdata.py

```diff
@@ -141,10 +179,21 @@ def write_lammpsdata(structure, filename, atom_style='full',
 
     # Convert coordinates to LJ units
     if unit_style == 'lj':
-        # Get sigma, mass, and epsilon conversions by finding maximum of each
-        sigma_conversion_factor = np.max([atom.sigma for atom in structure.atoms])
-        epsilon_conversion_factor = np.max([atom.epsilon for atom in structure.atoms])
-        mass_conversion_factor = np.max([atom.mass for atom in structure.atoms])
+        # If not specified by user, get sigma, mass, and epsilon conversions by finding maximum of each
+        if sigma_use_default:
+            sigma_conversion_factor = np.max([atom.sigma for atom in structure.atoms])
+            if sigma_conversion_factor == 0:
+                sigma_conversion_factor = 1
```

Comment:
We should at last warn the user that the conversion factors are getting changed if these conversion factors are zero.  Like:

`warnings.warn("sigma conversion factor cannot be inferred from the maximum sigma value in the ParmEd Structure.  Setting the sigma conversion factor to 1")`